bin/check-kube-pods-running.rb

```diff
@@ -59,6 +60,12 @@ class AllPodsAreRunning < Sensu::Plugins::Kubernetes::CLI
          long: '--pods',
          default: 'all'
 
+  option :not_ready_time,
+         description: 'Threshold for pods to be in the non ready state',
+         long: '--time TIME',
+         proc: proc(&:to_i),
+         default: 300
```

Comment:
If we wanted to make this backwards compatible we could set the default to `0` and then adjust it at a later time in a major release.